packages/rspack-cli/src/commands/serve.ts

```diff
@@ -89,8 +93,8 @@ export class ServeCommand implements RspackCommand {
           !compiler.platform.electron &&
           !compiler.platform.webworker;
 
-        if (isWebAppOnly) {
-          compiler.options.lazyCompilation ??= {
+        if (isWebAppOnly && userConfig.lazyCompilation === undefined) {
+          compiler.options.lazyCompilation = {
```

Comment:
Can we change the default `lazyCompilation` in `buildConfig`? So we do not need to expose extra methods.